docs/analysing-compilation-output.rst

```diff
@@ -0,0 +1,190 @@
+.. index:: analyse, asm
+
+#############################
+Analysing the Compiler Output
+#############################
+
+It is often useful to look at the assembly code generated by the compiler. However, the generated
+binary, i.e., the output of ``solc --bin contract.sol``, is difficult to read. It is recommended to use
+the flag ``--asm`` to analyse the assembly output.
+
+Consider the following contract (named, say ``contract.sol``):
+
+.. code-block:: Solidity
+
+    // SPDX-License-Identifier: GPL-3.0
+    pragma solidity >= 0.8.0 < 0.9.0;
```

Comment:
```suggestion
    pragma solidity ^0.8.0;
```

Reply:
Actually you can (and should) go back with the min version requirement a lot.

Reply:
Changed to 0.5.0